cob_hardware_config/cob4-4/config/Elmo.dcf

```diff
@@ -0,0 +1,4502 @@
+[FileInfo]
```

Comment:
can't we use the dcf file in the common directory? is there something special about cob4-4? if not, please use a softlink instead of copying